pulsar-client-shaded/pom.xml

```diff
@@ -217,6 +220,10 @@
                   <pattern>org.objenesis</pattern>
                   <shadedPattern>org.apache.pulsar.shade.org.objenesis</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>org.yaml</pattern>
+                  <shadedPattern>org.apache.pulsar.shade.org.yaml</shadedPattern>
+                </relocation>
```

Comment:
@jerrypeng We also need the same in `pulsar-client-admin-shaded` and `pulsar-client-kafka`

Reply:
I have shaded the dependencies in those modules as well